ExomeDepth/template.html

```diff
@@ -0,0 +1,116 @@
+<html>
```

Comment:
Even door een formatter halen:
https://www.freeformatter.com/html-formatter.html

Reply:
template is express geen HTML layout omdat elke spatie in de BED file terecht zou komen. Dat zou de BED file groot maken terwijl het niet nodig is.